docs/pages/guides/configuring-js-engines.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Configuring JS Engines
+sidebar_title: Configuring JS Engines
+description: A guide on configuring JS Engines for both Android and iOS in an Expo project.
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+JavaScript engines are responsible for executing your application code and providing various features like memory management, optimization, and error handling. Expo projects, by default, use [Hermes](https://hermesengine.dev/) as the JavaScript engine (in SDK 47 and lower, the default was JSC). It is also possible to switch to other engines like JSC or V8 for both Android and iOS platforms, but not for web because the JavaScript engine is included in the web browser.
```

Comment:
```suggestion
JavaScript engines execute your application code and provide various features such as memory management, optimization, and error handling. By default, Expo projects use [Hermes](https://hermesengine.dev/) as the JavaScript engine (in SDK 47 and lower, the default was [JavaScriptCore]((https://developer.apple.com/documentation/javascriptcore)). Switching to other engines, such as JSC or V8, for Android and iOS platforms is possible. However, not for the web because the JavaScript engine is included in the web browser.
```

Reply:
Changing from JSC to JavaScriptCore since this is the first time we are introducing that term on this page. 

Also, adding the link here because the term is referenced here for the first time. In general, that's a good practice.

Reply:
It would be also nice to add `JavaScriptCore` to our case police dict.

Reply:
Yes, great suggestion! I'll add it to the dict list.